plugin/index.php

```diff
@@ -19,13 +23,26 @@ function gutenberg_menu() {
 	);
 }
 
+// Hook.
+add_action( 'admin_enqueue_scripts', 'gutenberg_scripts_and_styles' );
+
+/**
+ * Scripts & Styles.
+ *
+ * @param  [type]    $hook
```

Comment:
It was not as clear to me, so I let it be to be reviewed and added at a later stage. I think the `$hook` is going to be a `string` value of screen name?